src/sentry/search/events/fields.py

```diff
@@ -2763,6 +2826,43 @@ def _resolve_user_misery_function(self, args: Mapping[str, str], alias: str) ->
             alias,
         )
 
+    def _resolve_percentile(
+        self, args: Mapping[str, str], alias: str, fixed_percentile: float = None
```

Comment:
I think the `Mapping` type might be incorrect- since the value can be either string, Column, SelectType, integer, float etc... ugh but I see that it's `Mapping[str, str]` everywhere else too. Maybe this can be cleaned up in a follow up.